src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py

```diff
@@ -274,30 +270,46 @@ def set_up(self):
         
     def body(self):
         s = self
-        src_cont = s.src_container
-        src_blob = s.src_blob
-        dst_cont = s.dest_container
-        dst_blob = s.dest_blob
         rg = s.resource_group
+
+        cont1 = 'cont1'
+        cont2 = 'cont2'
+
         _get_connection_string(self)
-        s.cmd('storage container create --name {} --fail-on-exist'.format(src_cont))
-        s.cmd('storage container create -n {} --fail-on-exist'.format(dst_cont))
+        s.cmd('storage container create --name {} --fail-on-exist'.format(cont1))
+        s.cmd('storage container create -n {} --fail-on-exist'.format(cont2))
+
+        blob_uri = 'blob1'
```

Comment:
It's really just the blob name for the blob I intend to test using the URI. The other is the blob name of the blob I test using the name components.